pandas/tests/indexes/common.py

```diff
@@ -721,3 +721,17 @@ def test_fillna(self):
                 expected[1] = True
                 self.assert_numpy_array_equal(idx._isnan, expected)
                 self.assertTrue(idx.hasnans)
+
```

Comment:
you need to iterate over all of the index types, see how the other tests do this. you are testing a single index class here, but we need to test them all.